lib/patient_crud.py

```diff
@@ -1,11 +1,7 @@
 from sqlalchemy.orm import Session
-
 from . import db_models
 
-
 def get_patient(db: Session, patient_id: int):
     return db.query(db_models.Patient).filter(db_models.Patient.id == patient_id).first()
-
 def get_patients(db: Session, skip: int = 0, limit: int = 100):
-    return db.query(db_models.Patient).offset(skip).limit(limit).all()
-
+    return db.query(db_models.Patient).offset(skip).limit(limit).all()
```

Comment:
not sure how patient_crud file was updated if you are not working on it. Did you delete an empty line here?